src/pages/help-centre/_help-articles.tsx

```diff
@@ -518,6 +518,162 @@ export const articles: ArcticlesType[] = [
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'fee-for-premium-agent',
+            },
+            {
+                title: (
+                    <Localize translate_text="What is the cost of using the premium payment agent’s services?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'cost-of-using-premium-agent',
+            },
+            {
+                title: (
+                    <Localize translate_text="Can a premium payment agent provide their services to another premium payment agent?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'provide-service-to-another-agent',
+            },
+            {
+                title: (
+                    <Localize translate_text="Can payment agent offer their services to a premium payment agent?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'offer-service-to-premium',
+            },
+            {
+                title: (
+                    <Localize translate_text="Can a payment agent offer their service to other payment agents?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'offer-service-to-agent',
+            },
+            {
+                title: (
+                    <Localize translate_text="Can all Deriv clients use premium payment agents to process transactions?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'all-clients-use-premium',
+            },
+            {
+                title: (
+                    <Localize translate_text="How can I add, remove or change my accepted payment methods?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'modify-payment-methods',
+            },
+            {
+                title: <Localize translate_text="Can I advertise my services to Deriv clients?" />,
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
+                label: 'advertise-to-clients',
+            },
+            {
+                title: (
+                    <Localize translate_text="Will I still be able to trade with my account after registering as a payment agent?" />
+                ),
+                category: 'Payment Agent',
+                sub_category: <Localize translate_text="Partners" />,
```

Comment:
Could u pls pass TString to this instead of Element and then update your component :pray: